codebuild/linux-clang3-x64/buildspec.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+#this buildspec assumes the ubuntu 14.04 trusty image 
+phases:
+  install:
+    commands:
+      - sudo apt-get update -y
+      - sudo apt-get update
+      - sudo apt-get install clang cmake3 cppcheck -y 
+  pre_build:
+    commands:
+      - export CC=clang
```

Comment:
Do these actually affect the subsequent build? I would expect that each line would be executed in a separate subshell.

Reply:
[Container] 2018/06/04 17:45:09 Running command cmake ../
-- The C compiler identification is Clang 3.4.0
-- The CXX compiler identification is Clang 3.4.0
-- Check for working C compiler: /usr/bin/clang
-- Check for working C compiler: /usr/bin/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/clang++
-- Check for working CXX compiler: /usr/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /codebuild/output/src039358197/src/github.com/awslabs/aws-c-common/build

it appears to.

Reply:
huh. interesting. okay then.